lib/imagemagick.js

```diff
@@ -137,6 +138,32 @@ function im_resize(options, force, callback) {
   var imcmd = sprintf('convert %s -resize "%sx%s"%s -quality %d %s', options.src, options.width, options.height, options.adjustment, options.quality, options.dst);
   exec(imcmd, function(err, stdout, stderr) {
     if (err) return callback(err);
+    if(options.overlay === 'play'){
+      im_add_play_button(options, force, callback);
+    } else {
+      im_info(options.dst, force, callback);
+    }
+  });
+}
+
+// add play button to an image
+function im_add_play_button(options, force, callback) {
+  // make sure we have a proper callback passed
+  if (typeof callback !== 'function') {
+    callback = force;
+    force = false;
+  }
+
+  // if it still not function, make a dummy function of it
+  if (typeof callback !== 'function') callback = function() {};
+
+  if (!options.dst) {
+    return callback(new Error('dst must not be ommited'));
+  }
+
+  var imcmd = sprintf('composite -gravity center -background none %s %s %s', playButtonPath, options.dst, options.dst);
```

Comment:
Yeah the last two args should both be options.dst. The last argument is the destination file and the second to last argument is the source file to be overlayed. We could have a temp file we write the intermediate-stage file to, but it becomes more complicated to run the overlay function within the resize function, especially when not all resizes also need overlays.